index.css

```diff
@@ -0,0 +1,167 @@
+*,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+li {
+  margin: 0px;
+  padding: 0px;
+}
+body {
+  font-family: Arial, sans-serif;
+  background: #f5f5f5;
+}
+
+#todo-app {
+  max-width: 913px;
+  max-height: 517px;
+  margin: auto;
+
+  background: white;
+  padding: 20px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  
+}
+#todo-app h1 {
+  font-family: "Inter", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 400;
+  font-style: normal;
+  font-variation-settings: "slnt" 0;
+  color: black;
+
+  font-size: 48px;
+
+  line-height: 58.09px;
+  text-align: left;
+}
+
+#todo-list {
+  list-style-type: none;
+  padding: 0;
+}
+
+.todo-item {
+  background-color: #ffea9e;
+  border: 1px solid #ddd;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  max-width: 380px;
+  min-height: 37px;
+  align-items: center;
+  padding-left: 10px;
+}
+.todo-item:nth-child(odd) {
+  background-color: #fca3a3;
+}
+.todo-item div:nth-of-type(2) {
+  display: flex;
+  flex-direction: column;
+}
+.todo-item div:nth-of-type(2) label {
+  font-family: Inter;
+  font-size: 10px;
```

Comment:
The font size is 12px, not 10px, you can view font information in Figma in the future: https://help.figma.com/hc/en-us/articles/360039956634-Explore-text-properties

-5